RecoJets/JetProducers/python/PileupJetID_cfi.py

```diff
@@ -9,6 +9,7 @@
 _chsalgos_5x = cms.VPSet(full_5x_chs,cutbased)
 _chsalgos_74x = cms.VPSet(full_74x_chs,cutbased)
 _chsalgos_76x = cms.VPSet(full_76x_chs,cutbased)
+_chsalgos_80x = cms.VPSet(full_80x_chs,cutbased)
 
 _stdalgos    = _chsalgos_76x
```

Comment:
I suppose, this means that the default behavior is unchanged.
This update just introduces selections for the new training/working point, which will need to be enabled when?